### PR TITLE
Leap 15 branding improvements

### PR DESCRIPTION
**Pull request by LelCP**
Woo, new branding update, what was improved this time:
- New YaST theme :o It was primarly made to work with sidebar which hopefully will come with update to yast-installation per [this discussion](https://github.com/yast/yast-installation/pull/657). It also works with header, in case it doesn't go through (but I believe in YaST team ;) ) and for Kubic.
- New and better XFCE splash (less fun, more serious :c)
- Bringing back Bold fonts in GRUB (in some places at least)
- Making gfxboot images all SVG and centering them out (I'm not sure how previous images got there, but they should be fine now ;) )
- Added logo to gfxboot background

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-NAME=leap
+NAME=tumbleweed
```
Comment (Vogtinator) [inline Makefile]: This doesn't look right.

Comment (LelCP) [reply Makefile]: Good thing you noticed, mindless copying and pasting is not the best idea ;)

**Comment by lnussel** [comment]
@Vogtinator  ok now?

**Comment by Vogtinator** [comment]
LGTM, but I didn't actually install it (yet).

**Pull request merged**